.ci/azure/android_arm64.yml

```diff
@@ -62,8 +62,23 @@ jobs:
     TMP_DIR: /mnt/tmp
     SHARE_DIR: /mount/cinfsshare/onnxtestdata
     CCACHE_DIR: $(SHARE_DIR)/ccache/master/android_arm64
+    LD_LIBRARY_PATH: $(Agent.ToolsDirectory)/Python/$(OV_PYTHON_VERSION)/x64/lib
+    OV_PYTHON_VERSION: 3.10.7
```

Comment:
see warnings https://dev.azure.com/openvinoci/dldt/_build/results?buildId=478842&view=results

Reply:
@jmiaskox is it something we can fix? I remember that there were some issues with `major.minor` version format before.

Reply:
Well this way we can specify exact version of Python, It is recommended to specify only major or major and minor version. Overall its only warning info that not should affect builds.